locale/en.po

```diff
@@ -2268,3 +2308,7 @@ msgstr ""
 "In order to reproduce you need to duplicate all of your organelles by "
 "collecting enough ammonia and phosphates.\n"
 "Look at the indicator in the bottom right to see how much more you need."
+
+#, fuzzy
+#~ msgid "CONFIRM_CAPITAL"
```

Comment:
What's with this translation being removed?

Reply:
Oh that, I changed the confirm button to use the capitalization but thought it will be nicer if it's in a PR that closes #1810 as well.